play/apps/leaderboard/models.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, *args, **kwargs):
     snake = models.ForeignKey(Snake, null=True, on_delete=models.CASCADE)
     mu = models.FloatField(null=True)
     sigma = models.FloatField(null=True)
+    unhealthy_counter = models.IntegerField(default=0)
```

Comment:
this is fine for our use case, but adding non-nullable fields will cause postgres to do a full table rewrite if postgres version < 11, we're currently running 10. The size of this table in battlesnake isn't very large, only 158 rows in production. See here https://docs.djangoproject.com/en/2.2/topics/migrations/#postgresql for more information.

Reply:
I understand. Good to know.

I can change the code so that the `unhealthy_counter` will be nulled and in the `increase_unhealthy_counter` function I do check for `None` at first. If it is `None` set it to 1, otherwise increase it.

Do you want me to do this? Should be not much work.